app/components/UI/AccountList/index.js

```diff
@@ -319,6 +327,23 @@ class AccountList extends PureComponent {
 			});
 	}
 
+	assignENSToAccounts = orderedAccounts => {
+		const { network } = this.props;
+		orderedAccounts.forEach(async account => {
+			try {
+				const ens = await doENSReverseLookup(account.address, network);
+				this.setState({
+					accountsENS: {
+						...this.state.accountsENS,
+						[account.address]: ens
+					}
+				});
```

Comment:
Fix: `setState` takes a function with `state, props` as arguments ([reference](https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous))
```suggestion
				this.setState(state => {
					accountsENS: {
						...state.accountsENS,
						[account.address]: ens
					}
				});
```

Reply:
Fixed!